src/Flow.jl

```diff
@@ -110,14 +128,14 @@ and the `AbstractPoisson` pressure solver to project the velocity onto an incomp
 @fastmath function mom_step!(a::Flow,b::AbstractPoisson)
     a.u⁰ .= a.u; scale_u!(a,0)
     # predictor u → u'
-    conv_diff!(a.f,a.u⁰,a.σ,ν=a.ν)
-    BDIM!(a); BC!(a.u,a.U,a.exitBC)
+    conv_diff!(a.f,a.u⁰,a.σ,ν=a.ν,perdir=a.perdir)
```

Comment:
These function arguments are getting pretty long... It's fine, I guess

Reply:
That's true, this will be resolved by `@generated` functions further down the line I suppose.